jump/src/context.rs

```diff
@@ -118,11 +118,20 @@ impl<'a> Context<'a> {
                 files_by_name.insert(key.as_str(), file);
             }
         }
+        let base = if let Ok(base) = env::var("SCIE_BASE") {
+            PathBuf::from(base)
+        } else if let Some(base) = &lift.base {
+            base.clone()
+        } else if let Some(dir) = dirs::cache_dir() {
```

Comment:
This is all the heavy lifting. I had already added the [`dirs`](https://crates.io/crates/dirs) crate a good while back but never actually used it.